podcasts/DownloadManager.swift

```diff
@@ -301,7 +301,7 @@ class DownloadManager: NSObject, FilePathProtocol {
             downloadAndStreamEpisodes.remove(downloadTaskUUID)
 
             if exportCompleted, let episode = dataManager.findBaseEpisode(uuid: episode.uuid) {
-                if let contentType = UTType.mpeg4Audio.preferredMIMEType {
+                if let contentType = episode.contentType {
```

Comment:
This may be overwritten by something according to https://github.com/Automattic/pocket-casts-ios/commit/37f1cd9. Probably need to test against that. Instead, I could store the contentType from above and reference here?